recipes/nnetsauce/meta.yml

```diff
@@ -0,0 +1,58 @@
+{% set name = "nnetsauce" %}
+{% set version = "0.8.3" %}
+
+package:
+    name: {{ name|lower }}
+    version: {{ version }}
+
+source:
+    url: https://github.com/Techtonique/{{ name }}/tree/master/dist/{{ name }}-{{ version }}.tar.gz
+    sha256: 2163e76d6edc6339207b8ee89275ab0fc9c8846f1f6a6997c9fed5628ad5ae0e
+
+build:  
+    number: 0  
+    script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+    build:    
+        - {{ compiler('c') }}
+        - {{ compiler('cxx') }}
+    host:
```

Comment:
It looks like this package might build C-extensions against numpy. Please see:

https://conda-forge.org/docs/maintainer/knowledge_base.html#building-against-numpy

Reply:
The package **uses Cython**. Either for **compiling a .pyx file**, or to **build some extensions** from cpp files. Maybe I should remove these compilers, since everything is done by Cython?

Reply:
Cython requires a c/c++ compiler to build the code, so you want to leave these in.

Reply:
ok